octoprint_mrbeam/__init__.py

```diff
@@ -1700,31 +1725,16 @@ def engraveCalibrationMarkers(self, intensity, feedrate):
     # 		return NO_CONTENT
 
     # Laser cutter profiles
-    @octoprint.plugin.BlueprintPlugin.route("/profiles", methods=["GET"])
+    @octoprint.plugin.BlueprintPlugin.route("/currentProfile", methods=["GET"])
     def laserCutterProfilesList(self):
```

Comment:
I think the name is for the old functionality. This is just a suggestion, just choose one following the convention if we have any:
```suggestion
    def get_current_laser_cutter_profile(self):
```

Reply:
On point. I will implement it.